[2023-03-20 16:13:05,916] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: to_posgrest_v4.4.create_postgres_table manual__2023-03-20T16:13:03.848432+00:00 [queued]>
[2023-03-20 16:13:05,963] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: to_posgrest_v4.4.create_postgres_table manual__2023-03-20T16:13:03.848432+00:00 [queued]>
[2023-03-20 16:13:05,966] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 16:13:05,967] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-03-20 16:13:05,968] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 16:13:06,008] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2023-03-20 16:13:03.848432+00:00
[2023-03-20 16:13:06,013] {standard_task_runner.py:52} INFO - Started process 13504 to run task
[2023-03-20 16:13:06,020] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'to_posgrest_v4.4', 'create_postgres_table', 'manual__2023-03-20T16:13:03.848432+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/to_postgresql.py', '--cfg-path', '/tmp/tmph25cqewu', '--error-file', '/tmp/tmp80rk8uvr']
[2023-03-20 16:13:06,023] {standard_task_runner.py:77} INFO - Job 93: Subtask create_postgres_table
[2023-03-20 16:13:06,309] {logging_mixin.py:109} INFO - Running <TaskInstance: to_posgrest_v4.4.create_postgres_table manual__2023-03-20T16:13:03.848432+00:00 [running]> on host 0b4cdb459521
[2023-03-20 16:13:06,619] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=sakabuana31
AIRFLOW_CTX_DAG_ID=to_posgrest_v4.4
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-20T16:13:03.848432+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-20T16:13:03.848432+00:00
[2023-03-20 16:13:06,697] {base.py:79} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: final, Login: ***, Password: ***, extra: {}
[2023-03-20 16:13:06,782] {dbapi.py:225} INFO - Running statement: 
            create table if not exists temperature (
                date DATE PRIMARY KEY,
                min DECIMAL(5, 2),
                max DECIMAL(5, 2),
                normal_min DECIMAL(5, 2),
                normal_max DECIMAL(5, 2),
                mean DECIMAL(5, 2),
                status_temp int
            ) 
        , parameters: None
[2023-03-20 16:13:06,787] {postgres.py:71} INFO - NOTICE:  relation "temperature" already exists, skipping

[2023-03-20 16:13:06,882] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=to_posgrest_v4.4, task_id=create_postgres_table, execution_date=20230320T161303, start_date=20230320T161305, end_date=20230320T161306
[2023-03-20 16:13:06,954] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-20 16:13:07,192] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
